src/lib/publish.js

```diff
@@ -38,7 +42,36 @@ async function publishBatch({ queue, messages, logger }) {
   }
 }
 
+/**
+ * publish a message to a notification topic
+ * @param {object} arg
+ * @param {string} arg.topic - topic to publish message to, e.g. an SNS ARN
+ * @param {string} arg.message - message to send
+ * @param {SNSClient} [arg.client] - SNS client to issue command to
+ * @param {boolean} [arg.catchSendErrors=true] - whether to catch-and-log unexpected errors when sending the message
+ * @param {Logger} logger - logger to use to log errors et al
+ * @returns {Promise<void>}
+ */
+async function notify({ client = snsClient, message, topic, catchSendErrors = true, logger }) {
```

Comment:
`catchSendErrors` is not needed, and in any case, the caller should handle the error

Reply:
@simone-sanfratello the caller can't handle the error if this function is swallowing it, which is why I wanted to provide the ability to opt out of the error-catching. The main reason I added the error catching was to be consistent with the other functions in here (which I took as your suggestion from [this comment](https://github.com/elastic-ipfs/indexer-lambda/pull/59#discussion_r931887867)), which swallow errors.

I'm going to remove the `catchSendErrors` option, and then have it swallow and log all errors (like the other functions in here). I think that's what you're suggesting but correct me if I'm wrong.